Dockerfile

```diff
@@ -1,17 +1,23 @@
 FROM centos:7
 MAINTAINER Trey Dockendorf <tdockendorf@osc.edu>
-RUN yum install -y centos-release-scl lsof sudo sqlite-devel
-RUN yum install -y https://yum.osc.edu/ondemand/1.6/ondemand-release-web-1.6-1.el7.noarch.rpm
-RUN yum install -y ondemand
+RUN yum install -y \
+        centos-release-scl \
+        https://yum.osc.edu/ondemand/1.6/ondemand-release-web-1.6-1.el7.noarch.rpm \
+    && yum clean all
```

Comment:
Multiple `yum clean all` will cause the next `yum` command to take longer.  Only really need one `yum clean all` after all other `yum` commands.